### PR TITLE
Curated playlists on website

### PR DESCRIPTION
**Pull request by kkuepper**
I also replaced Google Analytics with Application Insights

### CODE REVIEW

File: client/app/index.html [hunk 1]
```diff
@@ -66,6 +66,15 @@ <h2>We're sorry, it seems like your browser is out of date.</h2>
         </div>
     <![endif]-->
 
+    <script src="https://js.monitor.azure.com/scripts/b/ai.2.min.js"></script>
```
Comment (olivereisenhut) [inline client/app/index.html:69]: Is that script loading application insights which is used by the `angulartics`?

Comment (kkuepper) [reply client/app/index.html:69]: correct

File: common/app/scripts/services/_player.js [hunk 2]
```diff
@@ -271,6 +277,18 @@ angular.module('bmmLibApp')
...
         $(videoTarget).jPlayer('setMedia', src);
+
+        var status = $(videoTarget).data('jPlayer').status;
+        console.log("start playing", factory.id, factory.title, status, factory);
+        $analytics.eventTrack('Playback started', {
+          name: "Playback started",
+          PersonId: _api.getPersonId(),
+          track: factory.id,
+          title: factory.raw.title,
+          availability: "online",
```
[the remark this answers is not in the document]
Comment (kkuepper) [reply common/app/scripts/services/_player.js:288]: no. this is correct
image

File: common/app/scripts/services/_player.js [hunk 3]
```diff
@@ -271,6 +277,18 @@ angular.module('bmmLibApp')
       } else if (source.audio) {
         src = factory.resolveTypes(source.audios);
         $(videoTarget).jPlayer('setMedia', src);
+
+        var status = $(videoTarget).data('jPlayer').status;
+        console.log("start playing", factory.id, factory.title, status, factory);
```
[the remark this answers is not in the document]
Comment (kkuepper) [reply common/app/scripts/services/_player.js]: done

File: client/app/index.html (hunk 1 above)
Comment (kkuepper) [reply client/app/index.html:69]: I tried loading it from a npm package but ultimately failed and therefore decided to use the hosted version.

File: common/app/scripts/services/_player.js [hunk 4]
```diff
@@ -60,11 +60,22 @@ angular.module('bmmLibApp')
...
-          $analytics.eventTrack('play end', {
-            category: 'tracks',
-            label: factory.title,
-            value: factory.id
+          var status = $(videoTarget).data('jPlayer').status;
+          // we assume that the user has listened to the whole track. This is not correct but works good enough in most cases.
+          $analytics.eventTrack('Track played', {
+            PersonId: _api.getPersonId(),
+            albumId: factory.raw.parent_id,
+            availability: "Remote",
```
Comment (olivereisenhut) [inline common/app/scripts/services/_player.js:68]: this should be changed to online then?

**Pull request merged**